docs/react/devtools.md

```diff
@@ -123,7 +123,6 @@ function App() {
   const [showDevtools, setShowDevtools] = React.useState(false)
 
   React.useEffect(() => {
-    // @ts-ignore
```

Comment:
it's not unnecessary if you copy-paste this code into your typescript codebase. The error is:

```
TS2339: Property 'toggleDevtools' does not exist on type 'Window & typeof globalThis'.
```